Dockerfile

```diff
@@ -6,10 +6,11 @@ COPY . /opt/sapcli
 
 RUN cd /opt/sapcli &&\
 	python3 -m pip install -r requirements.txt &&\
-	chmod +x /opt/sapcli/sapcli &&\
-	echo '#!/bin/sh' > /usr/bin/sapcli &&\
-	echo 'python3  /opt/sapcli/sapcli "$@"' >> /usr/bin/sapcli &&\
-	chmod +x /usr/bin/sapcli
+	dos2unix /opt/sapcli/bin/sapcli &&\
```

Comment:
Why? If that's needed then change it in the repository but my vim claims the file bin/sapcli already has unix line endings (maybe it's a bit surprise but sapcli is written by people using Linux :).

Reply:
The command `dos2unix bin/sapcli` did no changes on my machine - that confirms it is not needed.

Reply:
Is it because you work on Windows and your git automatically changes line endings?